pkg/lockfile/parse-pdm-lock.go

```diff
@@ -0,0 +1,80 @@
+package lockfile
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/BurntSushi/toml"
+)
+
+type PdmLockPackage struct {
+	Name     string   `toml:"name"`
+	Version  string   `toml:"version"`
+	Groups   []string `toml:"groups"`
+	Revision string   `toml:"revision"`
+}
+
+type PdmLockFile struct {
+	Version  string           `toml:"lock-version"`
+	Packages []PdmLockPackage `toml:"package"`
+}
+
+const PdmEcosystem = PipEcosystem
+const PdmLockFileName = "pdm.lock"
+const pdmGroupDefault = "default"
+const pdmGroupDev = "dev"
+
+type PdmExtractor struct{}
```

Comment:
This matches the naming we use for the other extractors:

```suggestion
type PdmLockExtractor struct{}
```

Reply:
Changed the name